langchain4j-anthropic/src/test/java/dev/langchain4j/model/anthropic/internal/client/AnthropicEventSourceListenerTest.java

```diff
@@ -0,0 +1,72 @@
+package dev.langchain4j.model.anthropic.internal.client;
+
+import dev.langchain4j.agent.tool.ToolExecutionRequest;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.model.chat.TestStreamingResponseHandler;
+import dev.langchain4j.model.output.FinishReason;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.output.TokenUsage;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AnthropicEventSourceListenerTest {
+
+    @Test
+    void should_have_text() {
+        // given
+        TestStreamingResponseHandler<AiMessage> handler = new TestStreamingResponseHandler<>();
+        AnthropicEventSourceListener listener = new AnthropicEventSourceListener(handler, false);
+
+        // when
+        listener.onEvent(null, null, "message_start", "{\"type\":\"message_start\",\"message\":{\"id\":\"msg_018srTqYKTYLn7w1ZewrHoik\",\"type\":\"message\",\"role\":\"assistant\",\"model\":\"claude-3-5-sonnet-20240620\",\"content\":[],\"stop_reason\":null,\"stop_sequence\":null,\"usage\":{\"input_tokens\":451,\"output_tokens\":5}}           }");
```

Comment:
I am not sure this kind of test makes much sense as it will be outdated pretty soon...
I would suggest relying on integration tests that talk to real Anthropic API.